.env

```diff
@@ -1,7 +1,7 @@
 # Ports
-WARP_CORE_MYSQL_PORT=3307
+WARP_CORE_MYSQL_PORT=3308
```

Comment:
did you want to make this change to the .env?

Reply:
Tomas helped me set it up so I can work on Vulcan and warp-core. Can we change it back at the end?

Reply:
let's change this back before the merge